app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -1272,7 +1273,8 @@ def _validate_feature_config(self, data):
         for branch_data in data["treatment_branches"]:
             branch_error = None
             branch_warning = None
-            if branch_data.get("feature_enabled", False):
+            feature_enabled = branch_data.get("feature_enabled", False)
+            if feature_enabled:
```

Comment:
I still don't know what this change is accomplishing?

Reply:
sorry, forgot to get rid of that